_includes/getting-started.html

```diff
@@ -1,13 +1,42 @@
-<div class="anchor"><a name="getting-started"></a><h2>Run Meshery</h2></div>
+<!DOCTYPE html>
+<html>
+  <head>
+<style>
+  <style>
+.accordion {
+  
+  color: #444;
+  cursor: pointer;
+  padding: 18px;
+  width: 100%;
+  border: none;
+  text-align: left;
+  outline: none;
+  font-size: 15px;
+  transition: 0.4s;
+}
+
+
+.panel {
+  padding: 0 18px;
+  display: none;
+  overflow: hidden;
+}
+</style>
+</style>
+  </head>
+  <body>
+    <div class="anchor"><a name="getting-started"></a><h2>Run Meshery</h2></div>
 <div class="container flex">
 <div class="text editable code">
     <div class="post-content row" itemprop="articleBody">
         <h2>Step 1: Install and Start</h2>
         <!-- Docker -->
         <h3 style="margin-left:1em;">Bash user</h3>
-        <p style="margin-left:1.5em;">Install on <a href="https://meshery.layer5.io/docs/installation#quick-start">Mac or Linux</a> using Docker:</p style="margin-left:1em;">        
+        <p style="margin-left:1.5em;" class="accordion">Install on <a href="https://meshery.layer5.io/docs/installation#quick-start">Mac or Linux</a> using Docker:</p style="margin-left:1em;">        
```

Comment:
With the change to this section now being collapsed the hyperlink to the docs will need to be removed, so that users aren't navigated to the docs when they click to open show the instructions.